src/app/components/core/south/south-plugin-modal/south-plugin-modal.component.html

```diff
@@ -0,0 +1,37 @@
+<div id="south-service-plugin-modal" class="modal">
```

Comment:
While plugins are being fetched, please show “Fetching available plugins ...” and remove /hide once request is done.